CONTRIBUTING.md

```diff
@@ -1,16 +0,0 @@
-# Contributing
-
-## Participating in the Swift Evolution Process
-
-See [Participating in the Swift Evolution Process](https://swift.org/contributing/#participating-in-the-swift-evolution-process) on Swift.org and [process.md](process.md).
-
-## Contributing to the status page
-The [status page](https://apple.github.io/swift-evolution/) shows community activity related to the Swift Evolution Process. Changes to it should focus on supporting the efforts of the community in that process.
-
-Before making a pull request to change the status page, consider the following steps:
-                
-- **Socialize the idea**: Is there a broader desire in the Swift community for the feature? Ask. Ensure that the feature can be implemented using data from the Swift project itself, rather than depending externally derived data.
-  
-- **Develop a working copy**: Ideas are easiest to understand with a complete implementation. A quick prototype may be good enough for early stage feedback. Once the idea is understood, clean up the code, test it, and format it according to [JavaScript Standard](http://standardjs.com) style.
-  
-- **Request a review**: Initiate a pull request to the [swift-evolution repository](https://github.com/apple/swift-evolution) when a proposed change has a complete implementation. Include a link to a working copy, then assign the review to @krilnon. When everything looks good, the pull request will be merged.
```

Comment:
Was it intentional that the CONTRIBUTING page for Swift Evolution was deleted as part of this PR?  This looks like a mistake.

Reply:
The readme directs people to process.md, so I can see how it might make sense to remove contributing.md, given that it mostly points people to process.md once the status instructions are gone.